### DISCUSSION TITLE
Login Link

### DISCUSSION THREAD

**Discussion by AlineNap** [nl-design-system/backlog#240] [Component suggestions]
![Screenshot 2024-04-15 at 16 06 59](https://github.com/nl-design-system/backlog/assets/59806622/43bf3c67-f95f-463f-bd59-08656a25a1b3)

We willen graag meer leren over het Login Link component. In welke vorm, of vormen, wordt deze momenteel ingezet en toegepast binnen jouw organisatie?

Deel hieronder een link naar Figma, Storybook of screenshots. Met deze informatie kunnen we bewijzen dat het component onderdeel moet worden van de NL Design System component library.

Daarnaast wordt het delen van onderzoek gewaardeerd. Dus heb je zelf onderzoek gedaan en zijn er inzichten voor dit component? Dan horen we dit ook graag!

-------

## Naam
Login Link

## Doel
Biedt de mogelijkheid om naar de beschikbare inlogmiddelen te navigeren.

## Gerelateerde componenten
[Button](https://github.com/orgs/nl-design-system/discussions/205)

### Waarom een uniek component? 
Met een uniek component zijn updates vanuit het inlog patroon aan de copy, logo’s of opmaak gemakkelijk doorgevoerd.

## Richtlijnen
[Stijlhandleiding aansluiten Toegang - Logius ](https://www.logius.nl/domeinen/toegang/stijlhandleiding-aansluiten-toegang)
Kan uiteindelijk als patroon aangeboden worden.

## Onderzoek
Deze button is onderdeel een patroon, inloggen. Logius heeft de opdracht gekregen hier een uniforme klantreis voor te ontwerpen. Aanleiding hiervoor is de Wet digitale overheid waar één stelsel voor digitale toegang van burgers en bedrijven bij de overheid van kracht wordt. Onderdeel van de WDO is het toelaten van private authenticatiediensten en het uitbreiden van vertegenwoordiging opties, zoals ouderlijk gezag en bewindvoering. Mede door deze uitbreiding van het stelsel is het initiatief voor een uniforme klantreis opgezet.

Hiervoor zijn verschillende onderzoeken gedaan. Hierbij een paar rapporten:

- [Rapportage gebruikerstest - Ideaal inlogproces eID 28-07-2020.pdf](https://github.com/user-attachments/files/16929693/Rapportage.gebruikerstest.-.Ideaal.inlogproces.eID.28-07-2020.pdf)
- [Rapportage gebruikerstest - ideaal keuzescherm 02-02-2021.pdf](https://github.com/user-attachments/files/16929694/Rapportage.gebruikerstest.-.ideaal.keuzescherm.02-02-2021.pdf)
- [Rapportage gebruikerstest - uniforme klantreis 24-10-2022.pdf](https://github.com/user-attachments/files/16929695/Rapportage.gebruikerstest.-.uniforme.klantreis.24-10-2022.pdf)
- [Resultaten communicatietoets keuzescherm.pdf](https://github.com/user-attachments/files/16929696/Resultaten.communicatietoets.keuzescherm.pdf)

**Comment by AlineNap** [comment]
## Gemeente Den Haag 

**Login link**
- [Storybook](https://nl-design-system.github.io/denhaag/?path=/docs/css-cardauthentication--docs)
- [GitHub](https://github.com/nl-design-system/denhaag/tree/main/components/CardAuthentication)
- [Figma](https://www.figma.com/file/JpoY3waVoQGlLQzQXTL9nn/%F0%9F%8C%BB-Den-Haag-Design-System?type=design&node-id=10056-51382&mode=design&t=xltqqWpFHHPS4VB0-4)

----

### screenshots / varianten

**Apart inlogpagina**

<img width="1153" alt="Screenshot 2024-05-31 at 16 07 48" src="https://github.com/nl-design-system/backlog/assets/59806622/7e9ef9e8-4fdd-4c4d-9c72-8e8019cb4248">

**Onder dienst**

<img width="926" alt="Screenshot 2024-05-31 at 16 16 00" src="https://github.com/nl-design-system/backlog/assets/59806622/7694d56d-58c9-41df-b5be-9e23eb22c842">

-----

Notities

- label format: inloggen met [naam inlogmiddel] of de taak is beschreven zoals vergunning opzeggen.
- Aparte inlogpagina en inlog opties onder een dienst.

**Comment by AlineNap** [comment]
## Gemeente Utrecht

**Login link**
- [Storybook](https://nl-design-system.github.io/utrecht/storybook/?path=/docs/css_css-digid-button--docs)
- [GitHub](https://github.com/nl-design-system/utrecht/tree/main/components/digid-button)
- Figma niet gevonden

----

### screenshots / varianten

![image](https://github.com/nl-design-system/backlog/assets/59806622/5fb7c5df-f5a6-487f-9798-d76d87b949f9)


---- 

### notities
- label format: inloggen [naam inlogmiddel] of de taak is beschreven zoals kenteken aanpassen
- Inlog opties onder een dienst, geen aparte inlogpagina.

**Comment by AlineNap** [comment]
## Gemeente Amsterdam

**Login link**
Geen component gevonden in Storybook en Figma.

----

###screenshots / varianten

Op aparte inlogpagina: 
[link ](https://mijn.amsterdam.nl/?h=1&pk_vid=bc22777d291a88321718613123b17fc7)

<img width="449" alt="Screenshot 2024-05-31 at 17 07 03" src="https://github.com/nl-design-system/backlog/assets/59806622/a8807f79-6c31-4996-aa17-e553931d78de">

<img width="373" alt="Screenshot 2024-05-31 at 17 06 59" src="https://github.com/nl-design-system/backlog/assets/59806622/cea541c8-3c21-400f-b21c-6d8bf1b6a0a6">

Onder een dienst
[link](https://burgerzaken.amsterdam.nl/gaas-web/server/continue/StartVerhuizing#burgerzaken)

![Screenshot 2024-06-17 at 10 31 29](https://github.com/nl-design-system/backlog/assets/59806622/fbb5a446-165a-4cf6-a5dc-29d801587ed5)


**Belastingbalie Amsterdam**
[link](https://belastingbalie.amsterdam.nl/digid.info.php)

![Screenshot 2024-06-17 at 10 36 02](https://github.com/nl-design-system/backlog/assets/59806622/72bfd8c0-1e74-44ec-b79c-42251af18d6c)

**Mijn subsidies**
[link](https://mijnsubsidies.amsterdam.nl/regeling/625e9fd10e88c9ee793a0074)

![Screenshot 2024-06-17 at 10 41 30](https://github.com/nl-design-system/backlog/assets/59806622/8f8454d8-e374-4053-b0a1-b39bce0daa13)


-----

### Notities

- label format: inloggen met [naam inlogmiddel] of [naam inlogmiddel]
- Heeft een aparte inlogpagina en knoppen onder een dienst

**Comment by AlineNap** [comment]
## Logius

**Login link**
Storybook en Figma van huidige implementatie niet openbaar

- [Storybook nieuwe component](https://nl-design-system.github.io/lux/?path=/docs/web-components-login-link--docs)
- [GitHub](https://github.com/nl-design-system/lux/tree/main/packages/components-css/login-link)
- [Figma nieuwe component](https://www.figma.com/design/515tboh6xpSNziwQu6WyAy/LUX---UI-kit?m=auto&node-id=1125-2&t=Dn3iwYcrbwumOPYc-1)

----

### Screenshots / varianten

MijnOverheid:
<img width="257" alt="MijnOverheid2" src="https://github.com/nl-design-system/backlog/assets/59806622/37f599b2-1626-4880-b7b1-12727f733aef">

<img width="1266" alt="MijnOverheid" src="https://github.com/nl-design-system/backlog/assets/59806622/de333cac-4fc2-4dc3-be50-8c2662161a0f">



DigiD Machtigen
<img width="802" alt="DigiD" src="https://github.com/nl-design-system/backlog/assets/59806622/2f73e502-fb59-4d01-b570-d6d03b61ffb1">



DigiLevering
<img width="543" alt="Screenshot 2024-05-31 at 17 23 56" src="https://github.com/nl-design-system/backlog/assets/59806622/849b918c-8b99-4c1d-a724-9dff8997235c">

-----

### Notities

- label format: inloggen met [naam inlogmiddel]
- label format MijnOverheid: Log in met [naam inlogmiddel]

**Comment by AlineNap** [comment]
## UWV

**Login link**
? Storybook
? Figma

----

### Screenshots / varianten

![Screenshot 2024-06-17 at 10 53 58](https://github.com/nl-design-system/backlog/assets/59806622/68d3ad58-4f9e-445c-a998-19ff700b4a5f)

![Screenshot 2024-06-17 at 09 28 41](https://github.com/nl-design-system/backlog/assets/59806622/b3c112fc-6045-4fab-8eac-932eb7bb11d1)

![Screenshot 2024-06-17 at 09 29 02](https://github.com/nl-design-system/backlog/assets/59806622/453e078b-c101-405a-ba03-0b64e7786098)

![Screenshot 2024-06-17 at 09 29 22](https://github.com/nl-design-system/backlog/assets/59806622/aae9201d-1831-4f9f-ba9e-af9cebfcf7fc)

machtigen

![Screenshot 2024-06-17 at 10 54 25](https://github.com/nl-design-system/backlog/assets/59806622/f2e4446a-7194-4929-8dbc-2e2b459357cb)

In geheel
[link](https://mijn.uwv.nl/iam/inloggen.aspx)

![Screenshot 2024-06-17 at 10 55 22](https://github.com/nl-design-system/backlog/assets/59806622/e8e966e6-ca18-4867-a526-30673c2562cc)

-----

### Notities

- label format: gebruikt verschillende formats: Inloggen, [naam portaal], inloggen met eHerkenning
- Heeft een aparte inlogpagina, geen buttons gevonden onder een dienst. Wel een inlogknop van eHerkenning met het label: [naam portaal], deze staat in de start/home pagina.

**Comment by AlineNap** [comment]
## DSO, Omgevingsloket

**Login link**
Storybook: geen apart coponent
? Figma

----

### Screenshots / varianten
![Screenshot 2024-06-17 at 09 38 33](https://github.com/nl-design-system/backlog/assets/59806622/761b163d-6770-4d3b-824a-a0aa15823d0a)

![Screenshot 2024-06-17 at 09 38 42](https://github.com/nl-design-system/backlog/assets/59806622/40976d91-fd45-44e8-8f9b-3ebe1a254fec)

![Screenshot 2024-06-17 at 09 38 48](https://github.com/nl-design-system/backlog/assets/59806622/b2b1c1ed-9cb3-4048-9f3f-d92d0e1dfb0b)

-----

### Notities

- label format: inloggen met [naam dienst], Europeese login (voor eIDAS)
- Aparte inlogpagina, geen inlog opties gevonden onder een dienst.

**Comment by AlineNap** [comment]
## DUO

**Login link**
Geen component of documentatie gevonden op de documentatie website

----

### Screenshots / varianten

**Mijn DUO**
[link](https://mijn.duo.nl/isam/sps/MijnDUO-2-DigiDCC/saml20/logininitial?NameIdFormat=Transient&Target=https://mijn.duo.nl:443/particulier/portaal/dashboard/&&ForceAuthn=true&RequestBinding=HTTPPost&AuthnContextClassRef=urn:oasis:names:tc:SAML:2.0:ac:classes:PasswordProtectedTransport&AuthnContextComparison=minimum) 

![Screenshot 2024-06-17 at 10 04 44](https://github.com/nl-design-system/backlog/assets/59806622/b03b390a-e9d6-43cb-9f6d-d52059d30b61)

![Screenshot 2024-06-17 at 09 51 34](https://github.com/nl-design-system/backlog/assets/59806622/fbdb7a82-fc65-4938-97f4-7cd82cd8935c)

Machtigen
![Screenshot 2024-06-17 at 10 05 24](https://github.com/nl-design-system/backlog/assets/59806622/24f97367-e421-4ac4-be21-e8ea423d19dd)

**Mijn Inburgering**
[link](https://www.inburgeren.nl/inloggen-mijn-inburgering/) 

![Screenshot 2024-06-17 at 10 05 58](https://github.com/nl-design-system/backlog/assets/59806622/33afa568-ab5a-41c4-bcfc-9c7c42feee48)


**Onderwijs serviceregister**
[link](https://osr.kennisnet.nl)

![Screenshot 2024-06-17 at 09 58 26](https://github.com/nl-design-system/backlog/assets/59806622/12efafa7-d73d-41dc-8038-0f722319a934)

![Screenshot 2024-06-17 at 09 58 23](https://github.com/nl-design-system/backlog/assets/59806622/acadb662-1978-48c4-a41f-7803ab9fd519)


**Mijn helpdeskpo**
[link ](https://mijn.helpdeskpo.nl/Account/Login?ReturnUrl=%2f)

![Screenshot 2024-06-17 at 10 01 38](https://github.com/nl-design-system/backlog/assets/59806622/21429fee-65a6-4fda-a408-3fb45d191ad2)


**Mijn DUO zakelijk**
[link ](https://zakelijk.duo.nl/zakelijk/portaal/dashboard)

![Screenshot 2024-06-17 at 09 53 18](https://github.com/nl-design-system/backlog/assets/59806622/1baf32da-a94f-4d4f-8c12-13f59eb6e4c6)


**Personenregister kinderopvang**
[link](https://personenregisterkinderopvang.duo.nl)

![Screenshot 2024-06-17 at 09 52 29](https://github.com/nl-design-system/backlog/assets/59806622/8fb93f69-585f-4ebc-81cb-056d96166c08)

![Screenshot 2024-06-17 at 09 52 33](https://github.com/nl-design-system/backlog/assets/59806622/c252bed2-ee21-422f-9369-155ce1fd0c81)

![Screenshot 2024-06-17 at 09 52 37](https://github.com/nl-design-system/backlog/assets/59806622/adcc687f-a38c-43c1-81e9-da59984d116b)

Mijn Wft
[link](https://duo.nl/zakelijk/wet-op-het-financieel-toezicht/inloggen/)

![Screenshot 2024-06-17 at 10 11 10](https://github.com/nl-design-system/backlog/assets/59806622/45f4407e-5e15-4a5a-9cca-cd23adc21591)


-----

### Notities
- Label format: nloggen met [naam inlogmiddel], inloggen [naam portaal],
- Meestal aparte inlogpagina,
- DUO gebruikt meerdere patronen, vaak kom je nog op een aparte inlogpagina met keuze inlogmiddelen maar soms wordt dit ook overgeslagen en kom je direct bij eHerkenning.

**Comment by AlineNap** [comment]
## Open formulieren

**Login link**
[Storybook](https://open-formulieren.github.io/open-forms-sdk/?path=/docs/composites-login-options--docs)

----

### Screenshots / varianten

![Screenshot 2024-06-17 at 10 19 03](https://github.com/nl-design-system/backlog/assets/59806622/92522a00-0ef5-40dc-8448-89653bd68692)

![Screenshot 2024-06-17 at 10 18 38](https://github.com/nl-design-system/backlog/assets/59806622/f8070ead-afd0-43d4-b3e5-12b800969e30)


Machtigen

![Screenshot 2024-06-17 at 10 18 17](https://github.com/nl-design-system/backlog/assets/59806622/c348a088-734d-4d8a-8529-dc1761bf8afe)

![Screenshot 2024-06-17 at 10 18 45](https://github.com/nl-design-system/backlog/assets/59806622/ac95f905-d4fd-4cf9-9099-7f92e35a6bf6)


-----

### Notities
- twee label formats: [naam inlogmiddel], inloggen met [naam inlogmiddel]
- Per definitie staan deze knoppen onder een dienst gok ik? Gezien het om formulieren gaat. 
- Mogelijkheid om niet in te loggen wordt opgelost door een knop toe te voegen: begin form (anonymous)

**Comment by AlineNap** [comment]
## Rotterdam

**Login link**
Geen component gevonden

----

### Screenshots / varianten

Aparte inlogpagina
[link](https://am.rotterdam.nl/am/XUI/?realm=/alpha&authIndexType=service&authIndexValue=LoginComposite_Low&goto=https%3A%2F%2Fam.rotterdam.nl%3A443%2Fam%2Foauth2%2Fauthorize%3Fresponse_type%3Dcode%26client_id%3Dig-eso-client%26redirect_uri%3Dhttps%3A%2F%2Fconcern.ir.rotterdam.nl%2Fmijnloket%2Fid_token%2Fcallback%26scope%3Dopenid%2520profile%26state%3DcWluN2Rscmpha2Y1ZHZ6MjExMzl3bTQ1YmJucjhpMDovbWlqbmxva2V0%26acr_values%3Ddigid_eherkenning%3Alow%26prompt%3D%26max_age%3D60%26acr%3Ddigid_eherkenning%253Alow%26acr_sig%3D3tYY7N3s09KuUxlr1Mht0AP-gpsSWfbZMTqSIHz9ZLc#/)

![Screenshot 2024-06-17 at 10 49 12](https://github.com/nl-design-system/backlog/assets/59806622/9f257d3c-cf6a-47fe-a7d5-3336dd3d5c97)

-----

### Notities
- label format: inloggen met [naam inlogmiddel] als [burger, bedrijf/instelling],
- Aparte inlogpagina, geen knoppen onder een dienst gevonden.

**Comment by AlineNap** [comment]
## Belastingdienst

**Login link**
Bold, het design system van de Belastingdienst is niet openbaar

----

### Screenshots / varianten

![Screenshot 2024-06-17 at 11 06 20](https://github.com/nl-design-system/backlog/assets/59806622/64b57343-4ac3-4c5b-aff9-874275dfddf3)


Machtigen 

![Screenshot 2024-06-17 at 11 06 53](https://github.com/nl-design-system/backlog/assets/59806622/11ad39e2-f965-4a60-9727-0a19488d436e)

![Screenshot 2024-06-17 at 11 07 02](https://github.com/nl-design-system/backlog/assets/59806622/af6ec729-8c29-48a3-90aa-9164ac5c4398)

-----

### Notities

- label format: Inloggen met [naam inlogmiddel]
- Gebruik hetzelfde patroon voor de 4 verschillende portalen,[ Mijn Belastingdienst](https://mijn.belastingdienst.nl/GTService/#/inloggen), [Mijn toeslagen](https://gtmt.toeslagen.nl/tmx-aik-web/#/inloggen), [Voor ondernemers](https://mijnzakelijk.belastingdienst.nl/GTService/#/inloggen), [Voor douanezaken](https://mijndouane.belastingdienst.nl/GTService/#/inloggen)
- Geeft informatie over het aanvragen van een machtiging

**Comment by AlineNap** [comment]
## RVO

**Login link**
Geen component aanwezig

----

### Screenshots / varianten

**Aparte inlogpagina**
[link](https://mijn.rvo.nl/login?c=U3RhcnRVcmw9aHR0cHMlM0ElMkYlMkZtaWpuLnJ2by5ubCUyRmhvbWU%3D)

![Screenshot 2024-06-17 at 11 28 30](https://github.com/nl-design-system/backlog/assets/59806622/5d7f7605-d5bb-4d08-b4b5-2c9d78818572)

-----

### Notities
- Alleen aparte inlogpagina gevonden
- Label format: inloggen of log in voor Europees inloggen.  
- Extra inlogoptie.

**Comment by MariekeBrouwer** [comment]
![particulier](https://github.com/nl-design-system/backlog/assets/149144374/01710e7d-e76c-4904-b930-2305befa92d7)
![Desktop](https://github.com/nl-design-system/backlog/assets/149144374/cbbe6ac3-3af4-4c33-858a-7b3df55f0c72)

Voor OIP / MijnOmgeving zijn dit de recente designs rondom inloggen. Hierbij willen we dmv tabs een onderscheid maken tussen zakelijk/ondernemer.

**Comment by MrSkippy** [comment]
[MijnPGB](https://autorization.entreepgb.svb.nl/login/):

![image](https://github.com/user-attachments/assets/a0fb507f-c482-4155-8d5b-b9467aac2a22)
Heeft ook een login naar Vecozo.
en
![image](https://github.com/user-attachments/assets/0decdaaf-3308-4139-9d81-1f6c8f376e20)
Medewerkers naar nog veel meer.

**Comment by Rozerinay** [comment]
## ✨ Dit component is nu Help Wanted ✨

Help je mee hem door de Community stappen te krijgen?
[Login link op het Community bord](https://github.com/orgs/nl-design-system/projects/29/views/1?filterQuery=login+link)

**Comment by eslook** [comment]
## ✨ Dit component is nu Community ✨
Help je mee hem door de Candidate stappen te krijgen?
[Login Link op het Candidate bord](https://github.com/orgs/nl-design-system/projects/80/views/1?filterQuery=login+link)
